test/arrayops.jl

```diff
@@ -2342,4 +2342,4 @@ Base.view(::T25958, args...) = args
     @test t[end,1,end]   == @view(t[end,1,end])   == @views t[end,1,end]
     @test t[end,end,1]   == @view(t[end,end,1])   == @views t[end,end,1]
     @test t[end,end,end] == @view(t[end,end,end]) == @views t[end,end,end]
-end
+end
```

Comment:
This should be reverted.

Reply:
Woops, those damned ending newlines... Fixed.